documentation/developers.md

```diff
@@ -56,6 +55,31 @@ might be implemented to utilize the `compare-levels` function:
                                        :z "y"))}))
 ```
 
+## Exposing the /status endpoint
+
+If you want your registered status functions to be accessible via HTTP(S),
+you need to route the status service accordingly:
+
+```
+webserver: {
+  default: {
+    ssl-port: 9001
+    ssl-cert: /etc/ssl/certs/myhostname.pem
+    ssl-key: /etc/ssl/private_keys/myhostname.pem
+    ssl-ca-cert: /etc/ssl/certs/ca.pem
+    default-server: true
+  }
+}
+
+web-router-service: {
+  "puppetlabs.trapperkeeper.services.status.status-service/status-service": {
+    route: /status
+    server: default
+  }
+}
+```
+
+For information on HTTP serving of the status endpoint, see the [Status Proxy documentation](./status-proxy-service.md).
```

Comment:
I think this could probably be reworded to be a little clearer.  As it stands, my first thought is "well, I could just change the jetty configuration above to accept plaintext connections".  So maybe instead of "HTTP serving of the status endpoint" something like "enabling proxying of plaintext HTTP `/status` requests to an otherwise HTTPS-protected server" (or something less awkward).